content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx

```diff
@@ -669,3 +708,7 @@ Without `select` and `distinct`, the query would return:
   }
 ]
 ```
+
+```
```

Comment:
Is this empty?

Reply:
I keep getting blank sets of code marks at the end of pages. I don't know if they were there, or if my editor is adding them. I'm certainly not doing it manually.